tools/check-nfkc.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+# Checks if all normative and informative
+# WGSL spec sources are NFKC normal
+# Note: Filters sources by the working directory,
+# any other directory can benefit as well as WGSL
+
+import unicodedata
+import subprocess
+import sys
+
+files = subprocess.run(
+    ["git", "ls-files", "--recurse-submodules"], check=True, capture_output=True
+).stdout.splitlines()
+
+fix = "--fix" in [unicodedata.normalize("NFKC", i) for i in sys.argv]
```

Comment:
The normalization here is probably more than we need?
But I'm not harmed by it. :-)

Reply:
NFKC is much safer with cases like punctuation and international scripts and arguably might protect against a new class of potential entropy even in Latin characters: copy text from screenshot through OCR, which sometimes picks things like roman numeral code points instead of spelling it, which is something NFKC does effectively fix.